frontend/src/lib/taxonomy.tsx

```diff
@@ -1803,6 +1811,20 @@ for (const [key, value] of Object.entries(CORE_FILTER_DEFINITIONS_BY_GROUP.event
     }
 }
 
+for (const key of SESSION_PROPERTIES_ALSO_INCLUDED_IN_EVENTS) {
+    const mappedKey = key.replace(/^\$/, '') !== '$current_url' ? key : 'url'
+
+    if (key in CORE_FILTER_DEFINITIONS_BY_GROUP.event_properties) {
+        const eventProps = CORE_FILTER_DEFINITIONS_BY_GROUP.event_properties as Record<string, CoreFilterDefinition>
+
+        eventProps[`$session_entry_${mappedKey}`] = {
+            ...eventProps[key],
+            label: `Session entry ${eventProps[key].label}`,
+            description: `${eventProps[key].description}. Captured at the start of the session and remains constant for the duration of the session.`,
```

Comment:
Any suggestion here on the description would be great. I copied what we had commented on posthog-js.

I also avoided adding them together with session_properties because the key mapping is different.